README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/symfony/flex.svg?branch=master)](https://travis-ci.org/symfony/flex)
```

Comment:
In the past it was decided that Symfony repositories don't include these badges in their READMEs, so we should remove this. Thanks!